scripts/unzipEdge.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+{
+    unzip $1 src/out/Release/gen/devtools/* -d $2
+    unzip $1 src/out/Release/gen/devtools/*/* -d $2
```

Comment:
Could you open an issue to remove support for the old directory support and tag the code that needs to be removed with it?

Reply:
Created: https://github.com/microsoft/vscode-edge-devtools/issues/161

Will link in code once this PR merges